controlplane/kubeadm/internal/workload_cluster_coredns_test.go

```diff
@@ -82,7 +82,7 @@ func TestUpdateCoreDNS(t *testing.T) {
 				Spec: corev1.PodSpec{
 					Containers: []corev1.Container{{
```

Comment:
Please revert all those test cases. The logic shouldn't be changed (and isn't) the same applies to the test cases